cmd/ncproxy/ncproxygrpc/networkconfigproxy.proto

```diff
@@ -71,19 +72,27 @@ message PortNameEndpointPolicySetting {
 }
 
 message IovEndpointPolicySetting {
-    uint32 IovOffloadWeight = 1;
-    uint32 QueuePairsRequested = 2; 
-    uint32 InterruptModeration = 3;
+    uint32 iov_offload_weight = 1;
+    uint32 queue_pairs_requested = 2;
+    uint32 interrupt_moderation = 3;
+}
+
+message DnsSetting {
+    repeated string server_ip_addrs = 1;
+    string domain = 2;
+    repeated string search = 3;
 }
 
 message CreateEndpointRequest {
+    reserved 8 to 15;
```

Comment:
why these specifically or was it just a guess that we may want this range for policies?

Reply:
Just to reserve spots for future policies/fields